_topic_maps/_topic_map.yml

```diff
@@ -2152,11 +2118,6 @@ Topics:
     File: creating-machineset-nutanix
   - Name: Creating a compute machine set on OpenStack
     File: creating-machineset-osp
-  - Name: Creating a compute machine set on RHV
-    File: creating-machineset-rhv
-    Distros: openshift-enterprise
-  - Name: Creating a compute machine set on oVirt
-    File: creating-machineset-rhv
     Distros: openshift-origin
```

Comment:
You'll have to remove this line too, otherwise it's limiting the above OpenStack assembly to only OKD, which is what was causing a build failure from an xref